Dockerfile.template

```diff
@@ -8,7 +8,7 @@ RUN groupadd -r mysql && useradd -r -g mysql mysql
 RUN set -ex; \
 	apt-get update; \
 	if ! which gpg; then \
-		apt-get install -y --no-install-recommends gnupg; \
```

Comment:
gnupg2 is a dummy transitional package

https://packages.ubuntu.com/focal-updates/gnupg2

not sure why you changed this.

Reply:
Something wasn't OK during apt install. I always got installation error from the apt about unavailable gnupg package. I will check this again.

Reply:
Confirmed. It was only a temporary error for gnupg from apt servers. Docker build worked again.

Reply:
Was it this - dc4115360216763f5a8ad0c9e02d345974bf30de - race condition? or a flaky server keyserver.ubuntu.com? (in which case I probably need to check keys.opengpg.org).

ref: https://github.com/docker-library/official-images/#image-build

Reply:
Yesterday I have tried to build image using Podman and I got random errors from `apt-get` for different packages. Tried `podman build` 3-4 times, the last try worked for me. Today I got that the `gnupg` is not available (`unvailable package`?) from `docker build`. N times. After changing to `gnupg2` build success. Now it success after reverted changes. I think it was a flaky apt server or some bad internet connection?

Reply:
> I got random errors

If this is your description of the error message, the replies you get will also be random ;)

Reply:
@ottok The `docker build` command doesn't write output to log file. Docker Daemon also. No build output from `sudo journalctl --unit docker.service --since yesterday
`. And my terminal buffer length was too short to store this. I was not focusing on these errors but simple hit `docker build` several times with different errors for each invocation. I can only describe them as `random errors` :)